seq2science/rules/quantification.smk

```diff
@@ -272,41 +275,33 @@ elif  "scrna_seq" == get_workflow():
             input:
                 featurebarcodes=expand("{genome_dir}/{{assembly}}.tsv", **config)
             output:
-                directory(expand("{genome_dir}/{{assembly}}/index/kallistobus/kite/", **config)),
+                directory(expand("{genome_dir}/{{assembly}}/index/kallistobus", **config)),
```

Comment:
If I see this correctly, the `kallistobus_ref` index and the `kallistobus_ref_kite` index now share the same output directory. Furthermore, the downstream `kallistobus_count` uses this directory, no matter the content.

If you were to change the index type, snakemake has no way to see which index is in the directory. Is this intended, or did I miss something?

Reply:
I think you might be right, if someone would change the index type Snakemake would not be able to distinguish. I think that was the reason why I added it in the first place :)